apps/mobile/src/navigationTypes.ts

```diff
@@ -142,10 +142,9 @@ export type MarketplaceTabScreenProps<
 
 // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
 export type TradeChecklistStackParamsList = {
```

Comment:
Can we add chatId and focus on it in the TradeChecklist itself? It is not a good practice to assume the app is in certain state for route to work properly. It would be much better to have the route itself containing the key of the resouces. 

So instead of relying on `chatForTradeChecklistAtom` we just pass `chatId` into navigation and focus the chat there.